src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/entity/ChangeType.kt

```diff
@@ -8,4 +8,5 @@ enum class ChangeType {
   ADMISSION,
   MERGE,
   RESET_DAYS,
+  MOVE,
```

Comment:
Can this be named BOOKING_MOVE. Just so its clearer exactly what the event was.

Reply:
Code amended.